lib/utils/__tests__/narrowFixRange.test.mjs

```diff
@@ -306,6 +306,71 @@ describe('narrowFixRange', () => {
 				'a {--foo: abcabc}',
 			);
 		});
+		test('declaration with trailing semicolon', () => {
+			const node = postcss.parse('a { opacity: 45%; }');
```

Comment:
```suggestion
			const node = postcss.parse('a { opacity: 45%; }').first;
```

[suggestion] Use the first node (the `opacity` declaration) to better match real world use cases.

Reply:
I see this suggestion is incorrect.
It should be `.first.first`